.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   golang:
     docker:
-      - image: circleci/golang:1.13
+      - image: circleci/golang:1.14
```

Comment:
why specify a version? docker grabs latest build

Reply:
I think is a good approach to set controlled version, and update by hand to check if it works. I saw lots of places not recommended to use latest by default.